web/libs/editor/src/components/Timeline/Views/Frames/Frames.tsx

```diff
@@ -232,15 +256,19 @@ export const Frames: FC<TimelineViewProps> = ({
         }
 
         if (region) {
-          const [start, end] = frame > baseFrame ? [baseFrame, frame] : [frame, baseFrame];
-          region.setRanges([start, end]);
+          if (isInstant) {
```

Comment:
Potentially missed setting isInstant to `true`. At least I can't find it for now.